src/Honeycomb.OpenTelemetry/HoneycombOptions.cs

```diff
@@ -13,18 +13,17 @@ namespace Honeycomb.OpenTelemetry
     public class HoneycombOptions
     {
         private const string OtlpVersion = "0.16.0";
+        private const string OtelExporterOtlpProtcolHttp = "http/protobuf";
+        private const string OtelExporterOtlpProtcolGrpc = "grpc";
+        private const string OtelExporterHttpTracesPath = "/v1/traces";
+
+        private bool isHttp = false;
 
         /// <summary>
         /// Default service name if service name is not provided.
         /// </summary>
         internal static readonly string SDefaultServiceName = $"unknown_service:{System.Diagnostics.Process.GetCurrentProcess().ProcessName}";
-        private static readonly string SDefaultServiceVersion = "{unknown_service_version}";
-
-        private string _tracesApiKey;
-        private string _metricsApiKey;
-        private string _tracesDataset;
-        private string _tracesEndpoint;
-        private string _metricsEndpoint;
+        internal static readonly string SDefaultServiceVersion = "{unknown_service_version}";

```

Comment:
```suggestion
        internal static readonly string SDefaultServiceVersion = "unknown_service_version";
```
I know this wasn't changed in this PR, but I realized this shows up with the brackets which I don't think we want.